sphynx/python/install-dependencies.sh

```diff
@@ -1,13 +1,17 @@
 #!/bin/bash -xue
 # Install PyTorch Geometric.
```

Comment:
Pytorch is our only dependency? Maybe worth renaming this file? I mean, someone might innocently add something at the end and get surprised by it not working, due to the if at the start. Or maybe reverse the if. and put the torch related pip installs inside? Btw, why the if? Doesn't pip handle this anyways?

Reply:
> Pytorch is our only dependency? Maybe worth renaming this file? I mean, someone might innocently add something at the end and get surprised by it not working, due to the if at the start. Or maybe reverse the if. and put the torch related pip installs inside? Btw, why the if? Doesn't pip handle this anyways?

Without the `if` it takes 40 seconds to do nothing. We could probably use Pipenv or some magic to handle this better, but I'm just happy it works at all. 😅 

I've reversed the `if`, thanks for the suggestion!